assets/webauthn.js

```diff
@@ -13,17 +14,19 @@ document.addEventListener('DOMContentLoaded', function(){
 
     let attResp;
     try {
-      attResp = await startAttestation(await resp.json());
+      const jsonResponse = await resp.json();
+      if (jsonResponse.status === 500) {
+        handleError(jsonResponse.title, elemBeginRegister);
+      }
+      attResp = await startAttestation(jsonResponse);
     } catch (error) {
+      let errorText;
       if (error.name === 'InvalidStateError') {
```

Comment:
What emits this name? Should it be class based (instanceof) ?

Reply:
It's from the spec that browser implement: https://www.w3.org/TR/webauthn/#createCredential
I'm assuming the browser throws this error when the browser's API is called here:
https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/methods/startAttestation.ts#L37

I took it from the package's example: https://github.com/MasterKale/SimpleWebAuthn/blob/34935e25bb3b2c25d005c653670b9c517ff8b050/example/public/register/index.html#L46